service/context.tf

```diff
@@ -0,0 +1,59 @@
+module "this" {
+  source  = "cloudposse/label/null"
```

Comment:
This looks like a reference to a local path, which is not committed with these changes (so it would fail);
also if it meant to be a path to a third party, then similarly to @davidkelliott comment, we cannot use it unless it's an already security approved provider